docs/api-spec.md

```
@@ -236,8 +236,7 @@
   | `name`                                                                      | string                      | ✅   |
   | `icon`                                                                      | string                      | ✅   |
   | `sounds`                                                                    | array&lt;ModeSoundInput&gt; | ✅   |
-
-  - `ModeSoundInput`: `{ sound_id: integer (required), name?: string\|null }`
+  | - `ModeSoundInput`: `{ sound_id: integer (required), name?: string\|null }` |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 3; Actual: 1; Too few cells, row will be missing data

(MD056, table-column-count)